cmd/controller-manager/app/options/options.go

```diff
@@ -140,7 +142,17 @@ func (o *Options) AddFlags(flags *pflag.FlagSet, allControllers []string, disabl
 		&o.PrometheusQuantiles,
 		"prometheus-quantiles",
 		map[string]string{"0.5": "0.01", "0.95": "0.01", "0.99": "0.002"},
-		"prometheus summary objective quantiles",
+		"Prometheus summary objective quantiles",
+	)
+
+	flags.StringToStringVar(
+		&federatedcluster.VirtualNodeLabels,
+		"virtual-node-labels",
```

Comment:
should we make the name of the flag more general? this option can also be used to filter out other types of nodes as well

Reply:
yes, we should. the final node collection consists of schedulable nodes. so how about using unschedulableNodeLabels here？

Reply:
how about `--resource-aggregation-node-filter`? using the term "unschedulable" here might be confusing

Reply:
please take a look.